test/hitchhiker/tree/node/testing.cljc

```diff
@@ -1,8 +1,10 @@
 (ns hitchhiker.tree.node.testing
   (:require
-   [hitchhiker.tree.utils.async :as ha]
+   [#?(:clj hitchhiker.tree.utils.clojure.async
```

Comment:
If you name files with the same filename but file ending .clj and .cljs then the runtime should figure out itself which one to take and you would not have to distinguish in the requires. Am I wrong?

Reply:
Yes. Though the way our macros are we need to think this through some more.